pkg/spireentry/reconciler.go

```diff
@@ -213,9 +210,9 @@ func (r *entryReconciler) reconcile(ctx context.Context) {
 	}
 }
 
-func checkIgnoredNamespace(ignoredNamespaces []*regexp.Regexp, namespace string) bool {
+func isNamespaceIgnored(ignoredNamespaces []*regexp.Regexp, namespace string) bool {
 	for _, regex := range ignoredNamespaces {
-		if !(regex.FindString(namespace) == "") {
+		if !(regex.MatchString(namespace)) {
```

Comment:
This is now broken. isNamespaceIgnored should return true if the regex is a match.

Reply:
didn't realize the code broke--sorry! fixed now.